lib/MediaConnection.js

```diff
@@ -0,0 +1,57 @@
+const Utils = require('./Utils');
+const Grammar = require('./Grammar');
+const debugerror = require('debug')('JsSIP:ERROR:Socket');
+
+debugerror.log = console.warn.bind(console);
+
+/**
+ * Interface documentation: https://jssip.net/documentation/$last_version/api/mediaconnection/
+ *
+ * interface MediaConnection {
+ *
+ *  method createSdpOffer(options: RTCOfferOptions): Promise<RTCSessionDescriptionInit>
+ *  method createSdpAnswer(options: RTCOfferOptions): Promise<RTCSessionDescriptionInit>
+ *  method _setLocalSdp(sdp: RTCSessionDescriptionInit): Promise<void>
```

Comment:
```suggestion
 *  method setLocalSdp(sdp: RTCSessionDescriptionInit): Promise<void>
```

Reply:
Ok, but we leave 
`if (transportType === JsSIP_C.DTMF_TRANSPORT.RFC2833)`
the SIP-DTMF method in the code?

Reply:
yes :+1: